app/lib/actions/foreman_gutterball/content_reports/report_formatter.rb

```diff
@@ -0,0 +1,138 @@
+module Actions
```

Comment:
i'm not sure `app/lib/actions` is the best place for the `ReportFormatter`. `app/lib/actions/blah` is where dynflow actions go. I would argue maybe `app/lib/foreman_gutterball/formatters` or `lib/foreman_gutterball` would be a better place.